tardis/montecarlo/base.py

```diff
@@ -125,6 +125,12 @@ def spectrum_virtual(self):
                 self.montecarlo_virtual_luminosity)
 
     @property
+    def spectrum_integrated(self):
+        return self.integrator.calculate_spectrum(
+                self.spectrum_frequency[:-1],
+                1000)
```

Comment:
The hardcoded 1000 is the resolution of the p-grid, right? If so, I think we should make this also a configuration variable.